deployment/Makefile

```diff
@@ -68,6 +68,11 @@ collectstatic:
 	#We need to run collect static in the same context as the running
 	# uwsgi container it seems so I use docker exec here
 	@docker exec -t -i $(PROJECT_ID)_uwsgi_1 python manage.py collectstatic --noinput
+	#
+	# RM : this is needed to copy some library over
+	# I don't know a better way to do this
+	sudo cp ../django_project/feti/static/feti/js/ZeroClipboard.swf static/js/
```

Comment:
I need to put ZeroClipboard.swf in the same folder with ZeroClipboard.js.
It is working fine if I test locally because I already put it inside the feti/js folder.
But, in production mode, Pipeline compress and combine the js, into contrib.js and appjs.js so I need to put the swf in the same problem, and I don't know how.